.github/workflows/jobs.yml

```diff
@@ -284,7 +297,10 @@ jobs:
       - name: Run job
         run: |
           mkdir -p build && cd build
-          cmake -GNinja -DICB=ON -DEXAMPLES=ON -DMPI=ON ..
+          if [[ ${{ matrix.int64 }} == ON ]]; then
+            _blas_lib="-DBLAS_LIBRARIES=openblas_64"
+          fi
+          cmake -GNinja -DICB=ON -DEXAMPLES=ON -DMPI=ON -DINTERFACE64=${{ matrix.int64 }} ${_blas_lib} ..
```

Comment:
Is this using `flang`?

Reply:
It uses gfortran on UCRT64 and flang on CLANG64.
We have an alias/virtual package called `fc` which is gfortran on UCRT64 and flang on CLANG64.